Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    myfinance (1.6.4)
+    myfinance (1.6.5)
```

Comment:
`VERSION` change shouldn't be in the pull request. This will be done when releasing the new version.

Reply:
+1